test/sql/pragma/test_show_tables.cpp

```diff
@@ -0,0 +1,58 @@
+#include "catch.hpp"
+#include "test_helpers.hpp"
+
+using namespace duckdb;
+using namespace std;
+
+TEST_CASE("Test SHOW/DESCRIBE tables", "[pragma]") {
```

Comment:
I think this file was added by accident, is that correct?

Reply:
Correct. I'll remove it.